group_job.go

```
@@ -84,12 +84,13 @@ func (gj *groupJob[T, R]) Drain() error {
 	return nil
 }
 
-func (gj *groupJob[T, R]) Close() error {
+func (gj *groupJob[T, R]) close() error {
 	if err := gj.isCloseable(); err != nil {
 		return err
 	}
 
 	gj.wg.Done()
+	gj.Ack()
```

golangci-lint (1.64.8)

Error return value of `gj.Ack` is not checked

(errcheck)